src/test/java/hudson/plugins/ec2/win/WinConnectionWithFIPSTest.java

```diff
@@ -57,4 +49,12 @@ public void testSetShortPassword() {
         assertThrows(
                 IllegalArgumentException.class, () -> new WinConnection("", "alice", "0123", false).setUseHTTPS(false));
     }
+
+    /**
+     * WinConnection class cannot be instantiated in FIPS mode, an {@link IllegalArgumentException} is expected
+     */
+    @Test
+    public void testInstantiationWinConnection() {
```

Comment:
All the tests in this class seems to be testing the same thing now.

Reply:
I have removed them